src/components/SDKProvider/SDKProvider.tsx

```diff
@@ -14,7 +14,7 @@ import { ABSmartly, ABSmartlyContext, ContextOptionsType, ContextRequestType, SD
 type SDKProviderNoContext = {
     sdkOptions: SDKOptionsType;
     context?: never;
-    contextOptions: Record<string, any>;
+    contextOptions: { units: Record<string, any> };
```

Comment:
In regards to version bump/backwards compatibility, is this not a breaking change which would require a larger version bump?